src/dropdown/dropdown.ts

```diff
@@ -350,7 +364,7 @@ export class NgbDropdown implements OnInit, AfterContentInit, OnChanges, OnDestr
 			this._open = false;
 			this._resetContainer();
 			this._positioning.destroy();
-			this._zoneSubscription?.unsubscribe();
+			this._afterRenderRef?.destroy();
```

Comment:
This probably isn't necessary anymore as AfterRender should autodestroy with the DestroyRef. (If the injector is indeed a NodeInjector)

Reply:
@JeanMeche but it will be destroyed only when the directive is destroyed, and this is the close() method, which just closes the dropdown. The directive is still alive after this call, but the afterRender "effect" that was created when open() was called before (and that will be re-crezted the next time open() is called again) should be destroyed on close. Am I missing something?

Reply:
Thank you for the review BTW :-)